pkg/command/bundle.go

```diff
@@ -47,14 +51,39 @@ var _ Interface = (*BundleOptions)(nil)
 func (opts *BundleOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().String("bundle", "", "Where to publish the bundle.")
 	cmd.Flags().String("directory", ".", "The directory to bundle up.")
+
+	cmd.Flags().String("git-url", "", "The git URL to clone the source from if using git clone rather than a bundle image (e.g. if using mink inside a CI/CD pipeline).")
+	cmd.Flags().String("git-rev", "", "The git revision (branch, tag, SHA) to clone the source from if using git clone rather than a bundle image (e.g. if using mink inside a CI/CD pipeline).")
+	cmd.Flags().Bool("git-verbose", false, "If using git to clone the source enable verbose logging")
 }
 
 // Validate implements Interface
 func (opts *BundleOptions) Validate(cmd *cobra.Command, args []string) error {
 	viper.BindPFlags(cmd.Flags())
+
 	opts.ImageName = viper.GetString("bundle")
 	opts.Directory = viper.GetString("directory")
 
+	gitURL := viper.GetString("git-url")
+	if gitURL != "" {
+		if opts.GitLocation == nil {
+			opts.GitLocation = &source.GitLocation{}
+		}
+		opts.GitLocation.URL = gitURL
+		opts.GitLocation.Revision = viper.GetString("git-rev")
+		opts.GitLocation.Verbose = viper.GetBool("git-verbose")
+
+		// lets create a sample source bundle image tag...
+		var err error
+		opts.tag, err = name.NewTag("gcr.io/sample/source-bundle:latest", name.WeakValidation)
+		if err != nil {
+			return err
+		}
```

Comment:
I'm not sure what this is for? 😅

Reply:
I wasn't sure if it would be bad if opts.tag was empty ;) I guess lets leave it empty and see? :)